webapi/src/main/scala/org/knora/webapi/responders/admin/PermissionsResponderADM.scala

```diff
@@ -641,7 +624,7 @@ class PermissionsResponderADM(responderData: ResponderData) extends Responder(re
               requestingUser = KnoraSystemInstances.Users.SystemUser
             )
       result = ap match {
-                 case Some(ap) => permissionsmessages.AdministrativePermissionGetResponseADM(ap)
+                 case Some(ap) => { permissionsmessages.AdministrativePermissionGetResponseADM(ap) }
```

Comment:
```suggestion
                 case Some(ap) => permissionsmessages.AdministrativePermissionGetResponseADM(ap)
```
Previous version was OK I think.

Reply:
done